### PR TITLE
Ali Atayee (coding partner) : footer html structure defined in index.…

### PR DESCRIPTION
**Pull request by aliatayee**
Ali Atayee has developed the following parts:
 - Footer section structure defined in index.html
 - form input elements defined
 - background styles applied in style.css
Safar Ali has developed the following parts:
- The heading and the Paragraphs.
- The button at the end of the form.
- Resolved the linters errors.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -193,20 +201,47 @@ <h2>About <br> Myself </h2>
...
+            <div>
+                <input type="text" name="full-name" id="name" size="40" maxlength="30" placeholder="Full Name" required>
+            </div>
+            <div>
+                <input type="email" name="email" id="email" size="40" maxlength="30" placeholder="Email" required>
+            </div>
+            <div>
+                <textarea name="description" id="description" rows="3" cols="42" maxlength="30"
+                    placeholder="Write your message here" required></textarea>
+            </div>
```
Comment (Whoistolu) [inline index.html]: - [ ] [REQUIRED] Kindly note that one of the requirements states that the Message input has maxlength="500" attribute. Please implement this.

Comment (Whoistolu) [inline index.html]: - [ ] [REQUIRED] Please note that there is a white space surrounding the form which is not required. You have a margin on the body.
Your Implementation   |  Figma Design
:-------------------------:|:-------------------------:
Screenshot from 2022-02-16 17-22-45 | Screenshot from 2022-02-16 17-22-34

**Review by Whoistolu — CHANGES_REQUESTED** [review]
Hi Team,
@aliatayee  & @safar1212 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

- [ ] [REQUIRED] Kindly Update your README file by adding your partner as Author 2 and imputing his/her details. The README file is the file that gives visitors an idea of what your project is about.

- [ ]  _Check the inline comments under the review for sections to improve._

### Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

#### N/A

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (safar1212) [reply index.html]: @Whoistolu 
Hi there,
thank you for mentioning those. 
could you please help me regarding the width?
Which property should I use to fix this issue?
Looking forward to hearing from you.

Comment (BenMukebo) [reply index.html:239]: > * [ ]  [REQUIRED] Kindly note that one of the requirements states that the Message input has maxlength="500" attribute. Please implement this.

- [ ] Please consider making the changes the previous review mentioned above.

**Review by BenMukebo — CHANGES_REQUESTED** [review]
### ***Hi @aliatayee,*** 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._

- [ ] Please, I strongly recommend you make all changes the preview reviewer said please, check the comments below and make all of them then you can submit them for review.


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏 💻  

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @BenMukebo ) in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by micheaol — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 

Hi @aliatayee and @safar1212 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## TO HIGHLIGHT: 
- Your project is professional 💯 
- Your repo is professional 💯 
- Well structured files 💯 
- You set up linter files and no errors. 💯 

well-1

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**